airflow/security/permissions.py

```diff
@@ -32,6 +32,7 @@
 RESOURCE_DAG_CODE = "DAG Code"
 RESOURCE_DAG_RUN = "DAG Runs"
 RESOURCE_IMPORT_ERROR = "ImportError"
+RESOURCE_DAG_WARNING = "DagWarning"
```

Comment:
This should be sorted alphabetically. `DAG Warnings` is probably better? (I plan to rename ImportError to DAG Errors after this)